packages/odyssey-react-mui/src/theme/components.tsx

```diff
@@ -295,6 +299,10 @@ export const components: ThemeOptions["components"] = {
           ".MuiButton-startIcon": {
             margin: "0",
           },
+
+          ".MuiButton-endIcon": {
+            margin: "0",
+          },
```

Comment:
👍

Reply:
I've swapped this to use `buttonClasses` and it seems to be working great